elastalert/util.py

```diff
@@ -47,8 +47,7 @@ def lookup_es_key(lookup_dict, term):
         subkey = ''
 
         while subkeys:
-            subkey += subkeys[0]
-            subkeys = subkeys[1:]
+            subkey = subkeys.pop(0)
```

Comment:
Should be ``subkey += subkeys.pop(0)``.
For example, if subkeys is ['foo', 'bar'], we want to test 'foo' then 'foo.bar'.